docs/source/reference/difference.rst

```diff
@@ -27,8 +27,8 @@ This is Intel CPU result.
 Random methods support dtype argument
 -------------------------------------
 
-NumPy's random value generator does not support dtype option and it always returns a ``float32`` value.
-We support the option in CuPy because cuRAND, which is used in CuPy, supports any types of float values.
+NumPy's random value generator does not support a `dtype` argument and instead always returns a ``float32`` value.
+We support the option in CuPy because cuRAND, which is used in CuPy, supports any type of float values.
```

Comment:
There are other issues here.  Could you fix them?

`np.random` returns `float64`.

```
>>> np.random.randn(3).dtype
dtype('float64')
```

`cupy.random` does not support `float16` (because cuRAND does not).

```
>>> cupy.random.randn(dtype=np.float16)
TypeError: cupy.random only supports float32 and float64
```